exampleSite/config.yaml

```diff
@@ -71,7 +71,7 @@ params:
   editPrefix: https://github.com/unicef/inventory-hugo-theme/blob/main/exampleSite/content
 
   # Shown on all pages. When logo is empty, it will show your site title, set above.
-  logo: ""
+  logo: "images/unicef-logo.png"
```

Comment:
Unfortunately, we cannot use the UNICEF logo on this site. There are specific requirements around how the UNICEF brand can be used and I have not cleared those steps with the UNICEF Inventory theme site.

We could work around this and use a non-UNICEF logo in place here, but we need to find something openly licensed so we could legally use it on this site. I think it is simpler, for now, to omit a logo for the example site. We will always get to see both without and with a logo with the UNICEF toolkits which deploy the theme.

Reply:
Oh i don't know that we can't use the logo. I will use a custom made logo👍

Reply:
Resolved in commit 1f42c23b9bdf850505370e28efae7d3d15c899b1.